backend/libs/room-operations.ts

```diff
@@ -83,6 +83,37 @@ export const createRoom = async (
     }
 };
 
+export const updateRoom = async (
+    room: Room,
+    tableName: string,
+    dynamoDb: DocumentClient = new DocumentClient(),
```

Comment:
Cqn this use a variable that is initiated when the file is imported ? As is I feel like the client will be created even if we reuse the lambda